<?xml version="1.0"?>
<launch>
    <param name="show_mapper_occupancy_grid" type="bool" value = "true"/>
    <param name="show_mapper_delta_occupancy_grid" type="bool" value = "false"/>
    <node pkg="comp0037_mapper" name="mapper_node" type="mapperNode" output="screen"/>
</launch>
